templates/tickets/ticket.html

```diff
@@ -60,10 +66,9 @@ <h3>Add new message to the ticket</h3>
         </div> <!-- /#ticket-left -->
 
         <div id="ticket-right">
-            {% if perms.tickets.can_moderate %}
+            {% if perms.tickets.can_moderate and ticket.sound %}
```

Comment:
That's ok, but lets rename the header below which says "Ticket linked content" and the text "No linked content" displayed when there is no sound. I'd say "Ticket's sound" instead of "Ticket linked content"m and "This ticket has no assigned sound or the sound assigned to this ticket was deleted." instead of "No linked content".

(for some reason Github does not allow me to add this comment to the specific lines in code)